src/content/docs/data-apis/understand-data/event-data/default-events-reported-new-relic-products.mdx

```diff
@@ -13,9 +13,9 @@ redirects:
   - /docs/insights/event-data-sources/default-events/default-events-reported-new-relic-products
 ---
 
-New Relic products report different [types of data](/docs/using-new-relic/data/understand-data/new-relic-data-types). One type of data reported is [event data](/docs/using-new-relic/data/understand-data/new-relic-data-types#event-data). Events are displayed in UI charts and tables, and also made available for [querying](/docs/using-new-relic/data/understand-data/query-new-relic-data). To understand the types of data available, see [Data available via NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql#what-you-can-query).
+New Relic capabilities, like APM, browser monitoring, and mobile monitoring, report different [types of data](/docs/using-new-relic/data/understand-data/new-relic-data-types). One type of data reported is [event data](/docs/using-new-relic/data/understand-data/new-relic-data-types#event-data). Events are displayed in UI charts and tables, and also made available for [querying](/docs/using-new-relic/data/understand-data/query-new-relic-data). To understand the types of data available, see [Data available via NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql#what-you-can-query).
```

Comment:
The part has "types of data available" and "data available" in quick succession. Maybe the wording can be tweaked so that it doesn't repeat.